src/js/datadog.js

```diff
@@ -53,7 +53,7 @@ function initRum({ isForm }) {
     beforeSend(event, context) {
       // Add header/response context to api errors
       if (event.type === 'resource' && event.resource.type === 'fetch') {
-        if (context.response.status >= 400) {
+        if (context.response.status && context.response.status >= 400) {
```

Comment:
Configured this way, the header/response stuff won't be included if `context.response.status` is empty.

We could do this instead:

```js
if (!context.response.status || context.response.status >= 400) {}
```

If we want to include it for empty status API requests.